Use date instead of datetime

ttung opened the pull request:
### Description
datetime is probably too much precision, and introduces wonky timezone issues.  Simplify by changing them to `DATE`.

Depends on #62 

### Test plan
Run migration, check the db.

phoenixAja reviewed (approved):
i agree datetime is too intense haha

Commit:
Use date instead of datetime

### Description
datetime is probably too much precision, and introduces wonky timezone issues.  Simplify by changing them to `DATE`.

### Test plan
Run migration, check the db.
